content/blog/2018-03-13-weathercan.md

```diff
@@ -0,0 +1,500 @@
+           ECOZONE, ZONE_NAME) %>%
+  summarize(n_stations = length(unique(station_id)),
+            mean_temp = mean(mean_temp, na.rm = TRUE)) %>%
+  ungroup()
+```
+
+Finally, we can plot this as a map using `mapview`. In this manner all the data associated with each ecological region has been preserved. The image in this article is static, but if you run this yourself you'll get an interactive map and you can click on any of the regions below to get more information. Note also that the number of stations (`n_stations`) is also available.
+
+``` r
+mapview(MB, zcol = "mean_temp", legend = TRUE)
+```
+
+![](../../themes/ropensci/static/img/blog-images/2018-03-13-weathercan/mapview-1.png)
+
+<center>
+<img src = "/img/blog-images/2018-03-13-weathercan/mapview-1.png" style = "width:80%">
+</center>
+<p>
+Surprisingly Churchill, MB (the north-eastern, green area) was almost balmy compared to south-western Manitoba!
+
+Combining data in general
+-------------------------
+
+I hope these examples will help guide you in the many ways in which you can integrate `weathercan` data into other data sets. There are many different types of data to integrate, but generally, the same principals apply to merging `weathercan` data as to merging all data:
+
+-   Make sure your data is summarized to the appropriate level first (i.e. don't try to merge hourly data with yearly data)
+-   Make sure you join data by the correct columns (i.e. include your index columns as well as the appropriate time/date column)
+-   Often, you'll need to make an intermediate, `index`, data frame in which you link sites or observations to a `station_id`, this can then be used to link specific weather observations to your other measurements
+-   For spatial data, make sure your two data sets have the same CRS, and consider the `st_join()` function from the [`sf` package](http://r-spatial.github.io/sf/) to join them
+-   Make it easier on yourself by using [`tidyverse`](https://tidyverse.org), specifically the packages `dplyr` and `tidyr` ([R for Data Science](http://r4ds.had.co.nz) is a great reference)
+
+### A note about reproducibility
+
+I would also highly recommend documenting the process by which you join your data. This makes it easy for you to keep track of what you've done and makes your work reproducible.
+
+You can easily document this process by keeping an R script with all the coding steps and then using the "Knit" button in RStudio (or use [`Rmarkdown`](https://rmarkdown.rstudio.com) directly). If you like a more polished document, consider using `roxygen` comments `#'` rather than regular R comments `#` directly in your R script. `roxygen` comments allow you to write your comments in Markdown which is then converted to regular or marked up text in the html/pdf file. Your .R script would look something like this (including `devtools::session_info()` at the end spits out information on your version of R and any packages loaded):
```

Comment:
Try to minimize use of words like "easily" since for beginners, something like that won't be easy. You could omit the word in this case.

Reply:
Having this technical paragraph at the end that is about general R stuff and not about weathercan takes the focus away. Better to consider omitting it.